src/main.c

```diff
@@ -82,6 +85,49 @@ bool parse_rect(const char* arg, struct rect* rect)
     return true;
 }
 
+char** read_filelist(size_t* files_num)
+{
+    size_t buffer_len = PATH_MAX;
+    char* buffer = NULL;
+    ssize_t line_len;
+
+    *files_num = 0;
+    size_t files_max = 32;
```

Comment:
Well, I can put something like `#define INITIAL_ARRAY_SIZE 32` at the top, but the question will remain. The answer is usually "because the developer thought it's a good enough value" :-)